HunterPie.Core/Game/Rise/MHRGame.cs

```diff
@@ -187,6 +202,17 @@ private void GetDeathCounter()
         Deaths = deathCounter;
     }
 
+    [ScannableMethod]
+    private void GetQuestLife()
```

Comment:
No need to create a whole new method for it when there's already the Death scanning method.

https://github.com/HunterPie/HunterPie/blob/67608749a5d0d455455dbd6f43258cecd13cd4f2/HunterPie.Core/Game/Rise/MHRGame.cs#L194-L203